app/models/category.rb

```diff
@@ -1,3 +1,7 @@
 class Category < ApplicationRecord
+  default_scope -> { order(title: :ASC) }
```

Comment:
I don't know why, but I never seen that someone use 'default_sope') but it's ok, just comment)